schema/CVE_Record_Format.json

```
@@ -98,6 +98,11 @@
             "minLength": 1,
             "maxLength": 1024
         },
+        "semver-2.0.0-version": {
```

Since a PATCH is backwards compatible.
```suggestion
        "semver-2.0-version": {
```

---

I may be incorrect, but I don't see this definition referenced anywhere. Is this intentional?

---

I believe we've resolved this point in QWG discussions since this comment was posted. "2.0" is not a valid SemVer version, and it would be awkward to version our SemVer version type with a version which is not itself a valid SemVer version.